templates/_helpers.tpl

```diff
@@ -29,12 +29,12 @@ use .Values.agent.image.tag if set,
 otherwise take the first version (before '_') from .Chart.AppVersion
 */}}
 {{- define "datafy-agent.agentImageTag" -}}
-{{- if .Values.agent.image.tag }}
-{{ .Values.agent.image.tag }}
-{{- else }}
- {{- $parts := splitList "_" .Chart.AppVersion -}}
-  {{- index $parts 0 -}}
-{{- end }}
+{{- if .Values.agent.image.tag -}}
+{{- .Values.agent.image.tag -}}
+{{- else -}}
+{{- $parts := splitList "_" .Chart.AppVersion -}}
+{{- index $parts 0 -}}
+{{- end -}}
 {{- end -}}
```

Comment:
```suggestion
{{- define "datafy-agent.agentImageTag" -}}
{{- (default (split "_" .Chart.AppVersion)._2 .Values.agent.image.tag) -}}
{{- end -}}
```

Reply:
also, please do not explain what you are doing in the comment above.
explain why.
"what" is not the place for comments